static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -668,3 +668,75 @@ svg.main-menu__icon--social__svg {
 .no-text-transform {
     text-transform: none;
 }
+
+// Breathing space between header and content
+@include mq($until: tablet) {
```

Comment:
This bit was removed accidentally in this PR: 
https://github.com/guardian/frontend/pull/14519/files#diff-a6a8d02d22743e9a6dc47e407c359fb6L677

Just adding it back.